src/services/getCategories.js

```diff
@@ -0,0 +1,9 @@
+import axios from "axios";
+export const getCategories = async () => {
+  try {
+    const {data: {categories}} = await axios.get("/api/categories");
+    return categories;
+  } catch (error) {
+    console.log(error);
```

Comment:
You may return an object with data or error, so that you can show the error in view.

Reply:
Sure! its a good idea.